package.json

```diff
@@ -33,18 +33,21 @@
     "@rollup/plugin-json": "^4.1.0",
     "@rollup/plugin-node-resolve": "^8.4.0",
     "@types/cli-spinner": "^0.2.0",
+    "@types/jest": "^27.5.1",
```

Comment:
This is the latest version of `@types/jest` available ATM (note the lack of major version alignment)